frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -263,398 +269,105 @@ export function SurveyForm({ id }: { id: string }): JSX.Element {
 }
 
-export function SurveyView({ id }: { id: string }): JSX.Element {
-    const { survey, dataTableQuery, surveyLoading, surveyPlugin, installingPlugin } = useValues(surveyLogic)
-    // TODO: survey results logic
-    // const { surveyImpressionsCount, surveyStartedCount, surveyCompletedCount } = useValues(surveyResultsLogic)
-    const { editingSurvey, updateSurvey, launchSurvey, stopSurvey, archiveSurvey, installSurveyPlugin } =
-        useActions(surveyLogic)
-    const { deleteSurvey } = useActions(surveysLogic)
+export function SurveyReleaseSummary({
+    id,
+    survey,
+    targetingFlagFilters,
+}: {
+    id: string
+    survey: Survey | NewSurvey
+    targetingFlagFilters?: Pick<FeatureFlagFilters, 'groups'> | null
+}): JSX.Element {
     const { cohortsById } = useValues(cohortsModel)
-    const { editPlugin } = useActions(pluginsLogic)
-    const [setupModalIsOpen, setSetupModalIsOpen] = useState(false)
-    const { user } = useValues(userLogic)
-
-    const [tabKey, setTabKey] = useState(survey.start_date ? 'results' : 'overview')
-    useEffect(() => {
-        if (survey.start_date) {
-            setTabKey('results')
-        }
-    }, [survey.start_date])
-
     return (
-        <div>
-            {surveyLoading ? (
-                <LemonSkeleton />
-            ) : (
+        <div className="flex flex-col mt-2 gap-2">
+            <div>
+                {survey.linked_flag_id || survey.conditions?.url || survey.conditions?.selector || targetingFlagFilters
+                    ? 'This survey will be released to users who match all of the following:'
+                    : 'This survey will be released to everyone'}
+            </div>
+            {survey.linked_flag_id && (
+                <div className="flex flex-row font-medium gap-1">
+                    <span>Are part of the feature flag:</span>{' '}
+                    {id !== 'new' ? (
+                        survey.linked_flag?.id ? (
+                            <Link to={urls.featureFlag(survey.linked_flag?.id)}>{survey.linked_flag?.key}</Link>
+                        ) : null
+                    ) : (
+                        <FlagSelector value={survey.linked_flag_id} readOnly={true} onChange={() => {}} />
+                    )}
+                </div>
+            )}
+            {survey.conditions?.url && (
+                <div className="flex flex-row font-medium gap-1">
+                    <span>Are on a page with a url that contains:</span>{' '}
+                    <span className="simple-tag tag-light-blue text-primary-alt">{survey.conditions.url}</span>
+                </div>
+            )}
+            {survey.conditions?.selector && (
+                <div className="flex flex-row font-medium gap-1">
+                    <span>Are on a page with a selector that matches:</span>{' '}
+                    <span className="simple-tag tag-light-blue text-primary-alt">{survey.conditions.selector}</span>
+                </div>
+            )}
```

Comment:
I feel like the UI could be simplified and improved here by doing something like:

**Page conditions:**
    _URL contains_: `posthog.com`
    _Selector matches_: `.my-special-element`